server/db/schema.ts

```diff
@@ -48,6 +48,8 @@ export const jobPostingsTable = pgTable('job_postings', {
   }),
   isPublished: boolean('is_published').default(false).notNull(),
   totalApplicants: integer('total_applicants').default(0).notNull(),
+  validTill: date('valid_till'),
```

Comment:
also update schema in ~/shared

Reply:
Also, rebase and resolve merge conflicts

Reply:
@RishiDesai17  Delete migrations and generate new one, preferably, just create new branch

Reply:
@shivam-sharma7 @amandesai01 sure, will do

Reply:
@RishiDesai17 If you're facing any issue then let me know